docs/packaging_policy.md

```diff
@@ -108,29 +108,51 @@ for each combination. There are some particular cases for this general rule:
 ## Options
 
 Recipes can list any number of options with any meaning, and defaults are up to the recipe itself. The CI cannot enforce anything
-in this direction. However, there are a couple of options that have a special meaning for the CI:
+in this direction. However, there are a couple of options that have a special meaning for the CI.
 
-* `shared` (with values `True` or `False`). The CI inspects the recipe looking for this option. If it is found, it will
+### Predeinfed Options and Known Defaults
+
+ConanCenter supports many combinations, these are outline in the [supported configurations](supported_platforms_and_configurations.md) document for each platform. 
+
+* `shared` (with values `True` or `False`). The CI inspects the recipe looking for this option. The **default should be `shared=False`** and will
    generate all the configurations with values `shared=True` and `shared=False`.
 
-   > Note.- The CI applies `shared=True` only to the package being built, while every other requirement will use their defaults
-   > (typically `shared=False`). It's important to keep this in mind when trying to consume shared packages from ConanCenter
+   > **Note**: The CI applies `shared=True` only to the package being built, while every other requirement will. It's important to keep this in mind when trying to consume shared packages from ConanCenter.
+   > It's important to keep this in mind when trying to consume shared packages from ConanCenter
    > as their requirements were linked inside the shared library. See [FAQs](faqs.md#how-to-consume-a-graph-of-shared-libraries) for more information.
 
-* `header_only` (with values `True` or `False`). If the CI detects this option, it will generate all the configurations for the
-   value `header_only=False` and add one more configuration with `header_only=True`. **Only one
-   package** will be generated for `header_only=True`, so it is crucial that the package is actually a _header only_ library, with header files only (no libraries or executables inside).
+* `fPIC` (with values `True` or `False`). The **default should be `fPIC=True`** and will generate all the configurations with values `fPIC=True` and `fPIC=False`.
+  This option does not make sense on all the support configurations so it should be be removed.
+
+   ```python
+   def config_options(self):
+   if self.settings.os == "Windows":
+      del self.options.fPIC
+
+   def configure(self):
+      if self.options.shared:
+         try:
+             del self.options.fPIC
+         except Exception:
```

Comment:
that emits pycharm warnings, seems like considered a bad practice (exception is too broad)

Reply:
FYI, Conan 1.53 will bring the new method `rm_safe` (also for `settings`) so it'll be possible to avoid those try-except structures. It'll look like this:

```python
   def configure(self):
      if self.options.shared:
         self.options.rm_safe("fPIC")
```

Reply:
That will be amazing!

Reply:
Noisy?